### ISSUE TITLE
NTR: Request for S11 Flexion larva

### ISSUE THREAD

**Issue by nvsvocabs** [nvs-vocabs/S11#21]
<H2> Required </H2><p><b>Term name (PrefLabel)</b><p>Flexion larva <p><b>Definition</b><p> Developmental stage beginning with flexion of the notochord, development of the caudal fin and fin rays in majority of species, together with supporting bones and cartilages of the homoceral fin.<p><H2> Optional </H2><p><b>Sources/references</b><p> Ahlstrom, E.H. and O.P. Ball. (1954). Description of eggs and larvae of jack mackerel (Trachurus symmetricus) and distribution and abundance of larvae in 1950 and 1951. Fish. Bull. 56:209-245. || Kendall, Jr. A.W, Ahlstrom, E.H. and Moser, H.G. (1984). Early life history stages of fishes and their character, American Society of Ichthyologists and Herpetologists, Special publication 1: 11–22. || Mwaluma, J.M., Kaunda-Arara, B., and Strydom, N.A. (2014). A Guide to Commonly Occurring Larval Stages of Fishes in Kenyan Coastal Waters. WIOMSA Book Series No. 15. xvi. 73pp.<p>

**Comment by danibodc** [comment]
Suggested term:

**Term name (PrefLabel)**: flexion larvae
**Definition:** An ichthyoplankton life stage between the start of notochord flexion and until complete notochord flexion. This is concurrent with development of the caudal-fin rays and supporting skeletal elements.

**Comment by danibodc** [comment]
We have now created the following term:

S1189 - flexion larvae
A fish life stage between the start of notochord flexion and until complete notochord flexion. This is concurrent with development of the caudal-fin rays and supporting skeletal elements.

We chose 'fish' as opposed to 'icythoplankton' in the definition to match FishBase: https://www.fishbase.se/glossary/Glossary.php?q=flexion+larva&sc=is

Note that this will not become live on the NVS until ~0730UT tomorrow.
